test_all_mac.hxml

```diff
@@ -0,0 +1,36 @@
+-cmd echo 'js test'
```

Comment:
Is it not possible to reference hxml here? I admit I've never run this on a Mac. What's the difference? I see a lot of cd'ing into directories that doesn't happen on the "regular" hxml targets. Is it something to do with the path convention for running binaries/scripts?